course_content/notebooks/dask_intro.ipynb

```diff
@@ -4,9 +4,45 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# An Introduction to Dask\n",
+    "# An Introduction to Dask"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Course overview\n",
+    "\n",
+    "Dask is a flexible parallel computing library for Python. It allows dynamic scheduling of tasks for optimising computations and includes collections (extensions of arrays, lists and dataframes) for use with parallel data processing.\n",
+    "\n",
+    "As of Iris v2.0, dask has been integrated with Iris. This means you can utilise all of the benefits of dask while keeping the fileformat interoperability and other functionality that Iris provides."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Why does this course exist?\n",
+    "\n",
+    "This course exists to enable you to use dask and Iris together for more performant, memory efficient data processing by showing useful patterns for combining dask and Iris.\n",
+    "\n",
+    "#### Iris/dask integration\n",
+    "\n",
+    "Iris uses dask to defer data loading and processing by default in a number of situations, including loading and some mathematical, analytical and statistical operations. This course will introduce patterns for producing your own bespoke data processing pipelines that leverages Iris and dask functionality \n",
+    "\n",
+    "This course will introduce some patterns for using Iris and dask together to produce your own bespoke data prcessing pipelines that meet your analytical needs while maintaining deferred data.\n",
```

Comment:
This sentence is almost exactly the same as the previous one ... is that meant?

Reply:
Hm, looks like one of them got missed from being removed as I was writing this all - hence this still being a draft!